src/bootstrap/compile.rs

```diff
@@ -48,6 +48,18 @@ pub fn std(build: &Build, target: &str, compiler: &Compiler) {
     let out_dir = build.cargo_out(compiler, Mode::Libstd, target);
     build.clear_if_dirty(&out_dir, &build.compiler_path(compiler));
     let mut cargo = build.cargo(compiler, Mode::Libstd, target, "build");
+    std_cargo(build, compiler, target, &mut cargo);
+    run_cargo(build,
+              &mut cargo,
+              &libstd_stamp(build, &compiler, target));
+}
+
+/// Configures the `cargo` command with env vars and such to build the standard
+/// library.
+pub fn std_cargo(build: &Build,
```

Comment:
Oh I don't think we'd want to end up doing that because while the `compile.rs` invocations immediately just run Cargo the `check.rs` will add more args (like `--no-fail-fast`) and also maybe not abort the build if the test run fails.